integration/apiserver.go

```diff
@@ -42,6 +43,14 @@ type APIServer struct {
 	StartTimeout time.Duration
 	StopTimeout  time.Duration
 
+	// Out, Err specify where APIServer should write its StdOut, StdErr to.
+	//
+	// If not specified, these will be left as `nil` and the output of the
```

Comment:
This is slightly confusing.

> However, leaving either of those to `nil` is perfectly fine and will not result in any error.

Maybe because this sounds slightly over-dramatic :-)